### DISCUSSION TITLE
Questions about trigEngine

### DISCUSSION THREAD

**Discussion by just-nik** [L3MON4D3/LuaSnip#1274] [Q&A]
I want to know how to use trigEngine = "pattern" as nothing happens as mentioned in Doc.md . I plan to use it to crate some import snippet for eg: tcHome, where tc is trigger and Home is the thing I want to import. How can I achieve it?

**Comment by L3MON4D3** [comment]
Hi :)
Could you try
```lua
	s({trig="tc(%w+)", trigEngine="pattern"}, {
		l("import " .. l.LS_CAPTURE_1)
	})
```
It is rather minimal, but works on my end

**Reply by just-nik** [reply]
this doesn't work for me. I guess the issue is  on my end. most probably in nvim-cmp plugin as it being strict with snippet triggers or so
